xbmc/windowing/WindowingFactory.h

```diff
@@ -29,6 +29,9 @@
 #elif defined(TARGET_WINDOWS) && defined(HAS_DX)
 #include "windows/WinSystemWin32DX.h"
 
+#elif defined(TARGET_LINUX) && defined(HAVE_GBM) && defined(HAS_GLES)
```

Comment:
TARGET_LINUX is wrong for all ports here. Same issue again: LINUX is not a platform we build for. The actual targets are defined by GBM, X11, Mir, etc. Those are sufficient and only a single one of those must be defined for a build.